modules/gapi/perf/cpu/gapi_imgproc_perf_tests_fluid.cpp

```diff
@@ -58,7 +58,7 @@ INSTANTIATE_TEST_CASE_P(BlurPerfTestFluid, BlurPerfTest,
 INSTANTIATE_TEST_CASE_P(GaussianBlurPerfTestFluid, GaussianBlurPerfTest,
     Combine(Values(ToleranceFilter(1e-3f, 0.01).to_compare_f()),
             Values(CV_8UC1, CV_8UC3, CV_16UC1, CV_16SC1, CV_32FC1),
-            Values(3),                                     // TODO: add size=5, when kernel is ready
+            Values(3, 5),
```

Comment:
@elatkin 
As you can see here, I add numeral 5 to instantiation's parameters of performance tests for GaussianBlur. So now all performance tests run for 3x3 and 5x5 kernel sizes according to our test API.